web-client/public/js/graph.js

```diff
@@ -517,12 +517,12 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
                  if (x1 === x2 && y1 === y2) { // if self referential...
                      xOffsets = {
                          2 : 14, 3 : 15, 4 : 15, 5 : 15, 6 : 16, 7 : 16.5, 8 : 16.5,
-                         9 : 17, 10 : 17.5, 11 : 18, 12 : 19, 13 : 19.5, 14 : 16
+                         9 : 17, 10 : 17.5, 11 : 18, 12 : 19, 13 : 19.5, 14 : 20.5
                      };
                  } else {
                      xOffsets = {
                          2 : 13, 3 : 13, 4 : 13.5, 5 : 14, 6 : 15.5, 7 : 16.5, 8 : 17,
-                         9 : 16, 10 : 17, 11 : 17, 12 : 16, 13 : 18, 14 : 18
+                         9 : 16, 10 : 17, 11 : 17, 12 : 18, 13 : 18, 14 : 19
```

Comment:
Could this be the answer to addressing my question in https://github.com/dondi/GRNsight/issues/318#issuecomment-337346596? The numbers don’t seem to bear out though because it looks like there is only a 1-pixel difference between going from 8 to 9 vs. 9 to 10. Still, I noticed this change so it got me wondering.

Reply:
This was to center the edges.